editor/editor_inspector.h

```diff
@@ -397,7 +405,7 @@ class EditorInspectorArray : public EditorInspectorSection {
 	void set_undo_redo(UndoRedo *p_undo_redo);
 
 	void setup_with_move_element_function(Object *p_object, String p_label, const StringName &p_array_element_prefix, int p_page, const Color &p_bg_color, bool p_foldable);
-	void setup_with_count_property(Object *p_object, String p_label, const StringName &p_count_property, const StringName &p_array_element_prefix, int p_page, const Color &p_bg_color, bool p_foldable);
+	void setup_with_count_property(Object *p_object, String p_label, const StringName &p_count_property, const StringName &p_array_element_prefix, int p_page, const Color &p_bg_color, bool p_foldable, bool p_movable = true, bool p_numbered = false, int p_page_length = 5, const String &p_add_item_text = "", const String &p_swap_method = "");
```

Comment:
The `p_movable`, `p_numbered`, `p_page_length` and `p_add_item_text` should be added to `setup_with_move_element_function` too.

The `p_swap_method` should not be added there IMO, but likely to the other function.